src/components/Footer.js

```diff
@@ -325,31 +357,33 @@ const Footer = () => {
               })}
             </SocialIcons>
           </FooterTop>
-          {linkSections.map((section, idx) => {
-            return (
-              <LinkSection key={idx}>
-                <SectionHeader>
-                  <Translation id={section.title} />
-                </SectionHeader>
-                <List>
-                  {section.links
-                    .filter((link) => link.shouldDisplay)
-                    .map((link, linkIdx) => {
-                      return (
-                        <ListItem key={linkIdx}>
-                          <FooterLink
-                            to={link.to}
-                            isPartiallyActive={link.isPartiallyActive}
-                          >
-                            <Translation id={link.text} />
-                          </FooterLink>
-                        </ListItem>
-                      )
-                    })}
-                </List>
-              </LinkSection>
-            )
-          })}
+          <LinkGrid>
+            {linkSections.map((section, idx) => {
```

Comment:
This logic needs to be changed a bit. We now also need a `shouldDisplay` prop for each section, then a filter here to check if that prop evaluates to true. Reason being, some of them we should hide for languages.

e.g. check out the footer for Czech (content v1.0): https://deploy-preview-1738--ethereumorg.netlify.app/cs/
We should hide both Community & Enterprise sections since there's no links to display

e.g. check out the footer for German (content v1.1: https://deploy-preview-1738--ethereumorg.netlify.app/de/
We should hide the Community section since there's no links to display

So Enterprise should display for v1.1 or higher. Community should display for versions higher than v1.1.

Reply:
Totally agree... see f3c1ae3 below, should fix it right up.